src/zope/app/server/_testing.py

```diff
@@ -0,0 +1,21 @@
+import sys
```

Comment:
Can this live in the `tests` directory/package? It's not intended to be public (the leading underscore does indicate that) or used for anything other than local tests. 

I sometimes put code like this in `tests/__init__.py`.

Reply:
Sure, why not.